src/content/docs/kubernetes-pixie/kubernetes-integration/get-started/kubernetes-integration-compatibility-requirements.mdx

```diff
@@ -9,12 +9,32 @@ redirects:
   - /docs/integrations/kubernetes-integration/get-started/kubernetes-integration-compatibility-requirements
 ---
 
-[New Relic's Kubernetes integration](/docs/integrations/kubernetes-integration/get-started/introduction-kubernetes-integration) can be [installed directly](/docs/integrations/kubernetes-integration/installation/kubernetes-installation-configuration#install) on a server or VM, or through several [cloud platforms](/docs/integrations/kubernetes-integration/installation/kubernetes-installation-configuration#cloud-platforms), such as GKE, EKS, AKS, or OpenShift. Each has a different compatibility with our integration.
+The [Kubernetes integration](/docs/integrations/kubernetes-integration/get-started/introduction-kubernetes-integration) is compatible with many different platforms including GKE, EKS, AKS, OpenShift, and more. Each has a different compatibility with our integration. You can find more information in this page.
+
+## Requirements [#reqs]
+
+The New Relic Kubernetes integration has the following requirements:
+
+* A New Relic account. Don't have one? [Sign up for free](https://newrelic.com/signup). No credit card required.
+* Linux distribution [compatible with New Relic infrastructure agent](/docs/infrastructure/new-relic-infrastructure/getting-started/compatibility-requirements-new-relic-infrastructure#operating-systems).
+* [`kube-state-metrics`](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-state-metrics) running on the cluster.
+* When using CRI-O as the container runtime, the [processes](/attribute-dictionary/?event=ProcessSample) inside containers are not reported. Performance data is collected at the [container](/docs/integrations/kubernetes-integration/understand-use-data/find-use-your-kubernetes-data#container-data) level.
+* If you are using Helm, see the specific [compatibility and requirements](/docs/kubernetes-pixie/kubernetes-integration/installation/install-kubernetes-integration-using-helm/#compatibility). 
 
 <Callout variant="important">
-If you are using Openshift, you can also use `kubectl` most of the time but be careful that `kubectl` does not have commands like `oc login` or `oc adm`. You may need to use `oc` instead of `kubectl`.
+  `kube-state-metrics` v2 or higher is supported from integration version 3.6.0 or higher.
+  <br/>Install the Kubernetes integration up to version 3.5.0 if you are using `kube-state-metrics` 1.9.8 or lower.
+  <br/>
+  Check the `values.yaml` file if you're updating `kube-state-metrics` from v1.9.8 to v2 or higher because some variables may have changed:
+
+  - [KSM v1.9.8](https://github.com/kubernetes/kube-state-metrics/blob/0f5e4969f24bfa5f7c884f66d9d6fdf6f1921c16/charts/kube-state-metrics/values.yaml)
+  - [KSM v2 or higher](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-state-metrics/values.yaml)
 </Callout>
```

Comment:
I would not mention KSM this early in the journey. KSM is treated as a dependency and pulled in by our helm chart and installer automatically, so most users won't even know it exists.

Advanced users which want to bring their own KSM might be interested in this information, but I'd say this is a very advanced use case, at the same level as other avanced configs like control plane monitoring.

Reply:
I've mentioned KSM on the components page that I've modified. Check that page again and let me know if we should remove KSM from both pages. Thanks!

Reply:
I've removed it because @csongnr agrees with your comment.